Node/src/botbuilder-teams.d.ts

```diff
@@ -59,6 +59,8 @@ export interface ChannelInfo {
  *
  * @member {string} [obejctId] User Id
  *
+ * @member {string} [name] The givenName and the surname of the member
```

Comment:
@bilby91 Can you change this to "The display name for the member"
This aligns with the Graph API documentation. Thanks!